lib/grammers-client/src/types/attributes.rs

```diff
@@ -0,0 +1,58 @@
+// Copyright 2020 - developers of the `grammers` project.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// https://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+use grammers_tl_types as tl;
+
+pub enum Attribute {
+    Audio {
+        voice: bool,
```

Comment:
I'd go as far as separating this into another variant (why'd you provide album for a voice recording?)

Reply:
done